guides/v2.1/cloud/cdn/fastly-waf-service.md

```diff
@@ -6,26 +6,35 @@ functional_areas:
+{:.bs-callout .bs-callout-info}
+For additional information about maintaining PCI compliance for your {{ site.var.data.ece }} stores, see [Magento's Approach to PCI compliance ](https://magento.com/pci-compliance?_ga=2.43680032.406717297.1541547390-1020097258.1507670992).
 
+## How it works
 
-Once deployed, the WAF monitors web and admin traffic to identify any suspicious activity. It filters HTTP and HTTPS traffic (GET and POST requests) against the WAF ruleset and blocks traffic that is malicious or does not comply with specific rules. It can block SQL injection attacks, cross-site scripting attacks, and more.
+The Managed Cloud WAF integrates with Fastly and uses the cache logic within the Fastly CDN 
+service to filter traffic at the Fastly global nodes. We enable the WAF service in your production environment with a default WAF policy based on [ModSecurity Rules from Trustwave SpiderLabs](https://www.trustwave.com/Products/Application-Security/ModSecurity-Rules-and-Support/) and the OWASP Top Ten security threats.
 
+The Managed Cloud WAF filters HTTP and HTTPS traffic (GET and POST requests) against the WAF ruleset and blocks traffic that is malicious or does not comply with specific rules. The WAF service filters only origin-bound traffic that attempts to refresh the cache. As a result, we stop most attack traffic at the Fastly cache, protecting your origin traffic from malicious attacks. By processing only origin traffic, the Managed Cloud WAF preserves cache performance, introducing only an estimated 1.5 milliseconds (ms) to 20 ms of latency to every non-cached request.
+
+{:.bs-callout .bs-callout-info}
+If you notice performance issues and want to rule out WAF-related issues, submit a [Magento support ticket](https://support.magento.com/hc/en-us/articles/360000913794#submit-ticket) to investigate the problem.
 
 ### Enabling the WAF
 
-Fastly implements the managed web application firewall for {{ site.data.var.ece }} using the Fastly CDN service. You do not have to install or maintain any hardware or software. Before you can use the WAF, you must set up your Fastly service and configure Origin shielding so that 
-all external traffic routes through the Fastly service. See [Set up Fastly]({{ page.baseurl }}/cloud/cdn/configure-fastly.html).
+Fastly implements the managed web application firewall for {{ site.data.var.ece }} using the Fastly CDN service. You do not have to install or maintain any hardware or software. Before you can use the WAF, you must set up your Fastly service and configure Origin shielding so that all external traffic routes through the Fastly service. See [Set up Fastly]({{ page.baseurl }}/cloud/cdn/configure-fastly.html).
```

Comment:
I had a nagging question about enabling WAF and whether an account owner could toggle that on or off. When I read this section, the good stuff comes after this paragraph, which sends a user off to Set up Fastly. I came back a couple of times to see that Onboarding process helps to enable it, unless I have an older account. I would prefer to see this information first and see the qualifier (about setting up Fastly) in the How it works section. It seems interruptive here.